test/data/executor.start.yaml

```diff
@@ -1,3 +1,5 @@
+annotations:
+    beta.screwdriver.cd/executor: k8s
```

Comment:
Are we supposed to have some rules about the key of the annotations? We just assume the `executor-router`? knows how to parse it?